src/tests/JIT/HardwareIntrinsics/Arm/Shared/SveSimpleNoOpTest.template

```diff
@@ -0,0 +1,117 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Numerics;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.Intrinsics;
+using System.Runtime.Intrinsics.Arm;
+using Xunit;
+
+namespace JIT.HardwareIntrinsics.Arm
+{
+    public static partial class Program
+    {
+        [Fact]
+        public static void {TestName}()
+        {
+            var test = new CreateFalseMaskTest__{TestName}();
+
+            if (test.IsSupported)
+            {
+                // Validates basic functionality works
+                test.RunBasicScenario_CreateFalseMask();
+
+                // Validates calling via reflection works
+                // TODO-SVE: Enable once register allocation exists for predicates.
```

Comment:
@a74nh - can you remind me why this doesn't work without predicate register support?

Reply:
I copied this from another template without giving it a critical look. Will be happy to remove if @a74nh confirms that the comment isn't required here.

Reply:
I enabled this test for `SveLoadVector_int` and ran it. It passed. Not sure what the issues was now - was possibly due to missing mask handling (not predicates).

@mikabl-arm - could you uncomment this and remove the TODO, it should work.

We should re-enable all the others in a future PR.